src/babel-loader-regex-builder.ts

```diff
@@ -1,3 +1,3 @@
 export function getBabelLoaderIgnoreRegex(dependencies: string[]) {
-  return `[\\/]node_modules[\\/](?!(${dependencies.join('|')}))`
+  return `/[\\\\/]node_modules[\\\\/](?!(${dependencies.join('|')})[\\\\/])/`
```

Comment:
I think we should put a comment here because this is confusing xD
```suggestion
  // [\\\\/] is a bit confusing but what it's doing is matching either a
  // backslash or forwards slash. Forwards slashes don't need to be
  // escaped inside a character group, and we need to escape the
  // backslash twice because we're in a string, and in a regex.
  //
  // If you console.log the regex it'll actually turn into:
  // [\\/]
  return `/[\\\\/]node_modules[\\\\/](?!(${dependencies.join('|')})[\\\\/])/`
```

Reply:
Yeah definitely. In an intermediate step I had it as an actual RegExp instead of a string (you can of course console.log a RegExp, it will to `.toString()` on it, which will escape it again), but that was a bit too magical.